workitem/workitemtype_repository.go

```diff
@@ -26,6 +26,7 @@ type WorkItemTypeRepository interface {
 	CreateFromModel(ctx context.Context, model *WorkItemType) (*WorkItemType, error)
 	List(ctx context.Context, spaceID uuid.UUID, start *int, length *int) ([]WorkItemType, error)
 	ListPlannerItems(ctx context.Context, spaceID uuid.UUID) ([]WorkItemType, error)
+	LoadByID(ctx context.Context, id uuid.UUID) (*WorkItemType, error)
```

Comment:
@pranavgore09  okay, I gotta why you added it now. But I am wondering if we don't use it why we want it there...